src/frontend/pages/game.tsx

```diff
@@ -91,30 +103,50 @@ export default function Home(): ReactElement {
         <div className={styles.actionBar}>
           <div className={styles.actionBarLeft}>
             {/* TODO: both of these values will be given by an API/socket */}
-            <h4>You Are: </h4>
-            <h4>Current Coins: </h4>
+            {/* TODO: add some JS to change styling of the color based on what color it is  */}
+            <h4 className={styles.actionBarText}>
+              You Are: <span style={{ color: 'green' }}>{playerColor} </span>
+            </h4>
+            <h4 className={styles.actionBarText}>
+              Current Coins:{' '}
+              <span style={{ color: Colors.darkPurple }}>{playerCoins} </span>
+            </h4>
           </div>
           <div className={styles.actionBarMiddle}>
-            <button> Take</button>
-            {/* TODO: add socket send here */}
-          </div>
-          <div className={styles.actionBarRight}>
             <input
-              className={styles.inputBox}
               type="number"
               value={takeVal}
               onChange={(event) => {
                 inputOnChange(event.target.value);
```

Comment:
if this is disabled do we need an on change?

Reply:
we don't need it but can we keep it commented in case we need to bring it back in the future?

Reply:
it's too late, but I would have said completely delete it because the git history is always there and the logic isn't too ridiculous